src/ape/managers/converters.py

```diff
@@ -140,6 +140,28 @@ def convert(self, value: Union[List, Tuple]) -> Union[List, Tuple]:
 list_tuple_converter = ListTupleConverter(None, None, None)  # type: ignore
```

Comment:
@johnson2427  Notice the type ignore comment is required for these

Reply:
oooooooohhhhhhhhhhhhhhhhh

Reply:
Thank you!